<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmClientes</class>
 <widget class="QDialog" name="FrmClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Omega Sistemas Clientes</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salvar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_atualizar">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Atualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_localizar">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Localizar</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>470</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Clientes</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>63</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Endere√ßo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_nome">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>351</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_endereco">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>83</y>
       <width>421</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_fone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>116</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fone:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_relatorio">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>230</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Relatorio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
